### PR TITLE
Add HBN

### LINKED ISSUE #116: Add fetcher for the HBN data

### PR DESCRIPTION
**Pull request by richford**
Also add progress bar to the url download function. And add an `AFQDataset.from_study()` static method.

Resolves #116

### CODE REVIEW

**Comment by richford** [comment]
This is now running into the same version errors as #119. So I'll wait until we merge that and then rebase.

**Review by arokem — COMMENTED** [review]
Looks great! I wonder whether generalizing the `from_dataset` method would be a useful thing, or just over-complicated.

File: afqinsight/datasets.py [hunk 1]
```diff
@@ -584,6 +586,95 @@ def from_files(
...
+        if study.lower() not in downloaders.keys():
+            raise ValueError(
+                f"study must be one of {downloaders.keys()}. Got {study} instead."
+            )
+
+        download_kwargs = dict()
+        if verbose is not None:
+            download_kwargs["verbose"] = verbose
+
+        data_dir = downloaders[study.lower()](**download_kwargs)
```
[the remark this answers is not in the document]
Comment (richford) [reply afqinsight/datasets.py:655]: I think I need you to expound on this a bit. I don't quite understand what you're suggesting. We could talk about it in person. Or maybe I can come back to this after I've had more caffeine.

[the remark this answers is not in the document]
Comment (arokem) [reply afqinsight/datasets.py:655]: After an in-person conversation, we came to the conclusion that this is probably not worth it. We anticipate this to be useful primarily in the context of datasets that get encoded explicitly in AFQ-Insight (e.g., in examples).

**Pull request merged**